src/services/config-store.js

```diff
@@ -63,6 +72,10 @@ class ConfigStore {
       this.logger.warn(`Your configDir ("${this.configDir}") does not exist. Please make sure it is set correctly.`);
     }
 
+    if (!this.doesSchemaDirExist()) {
+      this.logger.warn(`Your schemaDir ("${this.schemaDir}") does not exist. Please make sure it is set correctly.`);
```

Comment:
I suggest here to throw an error directly to avoid crashes
```suggestion
      throw new Error(`Your schemaDir ("${this.schemaDir}") does not exist. Please make sure it is set correctly.`);
```

Reply:
Do we want to stop the initialisation of a server with an in-app integration because of an agent misconfiguration?
I guess that's what `throw` will do.